ccpp/data/GFS_typedefs.F90

```diff
@@ -1122,7 +1121,7 @@ module GFS_typedefs
     logical              :: ca_sgs          !< switch for sgs ca
     logical              :: ca_global       !< switch for global ca
     logical              :: ca_smooth       !< switch for gaussian spatial filter
-    integer              :: iseed_ca        !< seed for random number generation in ca scheme
+    integer*8            :: iseed_ca        !< seed for random number generation in ca scheme
```

Comment:
I think we need to be careful here. What is the type / kind of this variable in `GFS_typedefs.meta`? I think we should use a proper kind variable and, if this variable ever gets used in CCPP, we should set type and kind accordingly in the metadata, too. Maybe we should do this right away to not forget about it? It doesn't cause a problem right now the way it is written. What do you think?

Reply:
I switched this to kind_dbl_prec in GFS_typdefs, and in stochastic_physisics